src/iuse.cpp

```diff
@@ -160,13 +160,13 @@ void remove_double_plut_mod( item &it, player &p )
     }
     p.add_msg_if_player( _( "You remove the double plutonium capacity mod of your %s!" ),
                          it.tname().c_str() );
-    item mod( "double_plutonium_core", calendar::turn );
+    item mod( "double_plutonium_core" );
     p.i_add_or_drop( mod, 1 );
     it.item_tags.erase( "DOUBLE_AMMO" );
     it.item_tags.erase( "DOUBLE_REACTOR" );
     // Easier to remove all cells than to check for the actual real maximum
     if( it.charges > 0 ) {
-        item cells( "plut_cell", calendar::turn );
+        item cells( "plut_cell" );
         cells.charges = it.charges / 500;
```

Comment:
Same here (and throughout). `charges` is an implementation detail of `item` so setting it directly violates encapsulation. Once all such usages are gone we can make it private to `item`